portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.data;
+
+/** An item on a todo list. */
+public final class Comments {
+
+  private final String text;
+  private final String text2;
+  private final String email;
+  private final String phone;
+  private final String message;
+
+  public Comments(String text, String text2, String email, String phone, String message) {
+    this.text = text;
```

Comment:
with two text fields, might be better to make variable names more descriptive

Reply:
Okay I will change it to what they are to make it more descriptive. Thanks